packages/app-store/routing-forms/lib/getSerializableForm.ts

```diff
@@ -67,23 +106,47 @@ export async function getSerializableForm<TForm extends App_RoutingForms_Form>({
 
   let teamMembers: SerializableFormTeamMembers[] = [];
   if (form.teamId) {
-    teamMembers = await prisma.user.findMany({
-      where: {
-        teams: {
-          some: {
-            teamId: form.teamId,
-            accepted: true,
+    teamMembers = normalizeTeamMembers(
+      await prisma.attributeToUser.findMany({
```

Comment:
This query controls the members and attributes logic. where->member->user->teams->some->teamId=<num>&accepted=true.